ramls/types/providers/providerDataAttributes.json

```diff
@@ -28,13 +28,18 @@
     },
     "supportsCustomPackages": {
       "type": "boolean",
-      "description": "provider token",
+      "description": "Whether this provider supports custom packages",
       "example": false
     },
     "proxy": {
       "type": "object",
       "description": "Proxy",
       "$ref": "../proxy.json"
+    },
+    "tags": {
+      "type": "object",
+      "description": "Provider tags",
+      "$ref": "../../raml-util/schemas/tags.schema"
```

Comment:
Just 1 clarification -- although tags are defined as part of attributes for provider here, we are populating values ONLY for get provider by id at this point (correct)? Hopefully this will not be any issue for UI as missing attributes will NOT be returned for list request.

Reply:
providerDataAttributes.json is used for retrieving provider by id and for Put provider

Reply:
Ok, thanks @natali-zaitseva -- just wanted to confirm that the added attribute does not cause issues for other methods which may be sharing the same definition (providerDataAttributes.json )